map_resolution: 0.05
world_frame: "odom"
pointcloud_topic_queue_length: 10
pointcloud_queue_max_wait_for_transform_s: 1.0
map_visualization_period_s: 1.0
map_autosave_period_s: -1.0
